docs/content/content/content-styles.mdx

```diff
@@ -638,7 +636,7 @@ Don’t use ‘such as’ or ‘includes’ in place of ‘for example’.
 
 For more information, see ‘e.g.’ or ‘for example’ in the DAFF Style guide (internal DAFF staff only).
 
-<ExampleContent heading={<H3>Example</H3>}>
+<ExampleContent heading={<H3>‘For example’ examples</H3>}>
```

Comment:
Does the Example heading below need changing?

Reply:
Oh yeah, these two examples need to be in a list.

- [ ] list for 'for example' examples